lang/gl/texts/contribute.html

```diff
@@ -103,8 +103,8 @@ <h3>Construíndo Comunidade</h3>
 		<h3>Comunicación</h3>
 		<p>Non sabemos moi ben que escribir aquí, podería axudarnos?</p>
 		<p>Xa en serio, hai moitas cousas que usuarias, contribuíntes e reutilizadoras fan con Open Food Facts,
-estaría moi ben que máis xente o soubese. En particular, precisamos axuda coas 
-<a href="/press">relacións cos medios</a> e públicas.</p>
+estaría moi ben que máis xente o soubese. In particular, we need help for public
```

Comment:
Any idea why this was un-translated? Was the source changed?

Reply:
I think a carriage return was introduced in the source, and everything got untranslated :-(

Reply:
nope, a single comma after "in particular,"